src/tests/data/restore.test.ts

```diff
@@ -483,4 +484,55 @@ describe("restore", () => {
     );
     expect(restoredData.appState.name).toBe(stubImportedAppState.name);
   });
+
+  it("bump versions of local duplicate elements when supplied (restore)", () => {
+    const rectangle = API.createElement({ type: "rectangle" });
+    const ellipse = API.createElement({ type: "ellipse" });
+
+    const rectangle_modified = newElementWith(rectangle, { isDeleted: true });
+
+    const restoredData = restore.restore(
+      { elements: [rectangle, ellipse] },
+      null,
+      [rectangle_modified],
+    );
+
+    expect(restoredData.elements[0].id).toBe(rectangle.id);
+    expect(restoredData.elements[0].versionNonce).not.toBe(
+      rectangle.versionNonce,
+    );
+    expect(restoredData.elements).toEqual([
+      expect.objectContaining({ version: rectangle_modified.version + 1 }),
+      expect.objectContaining({
+        id: ellipse.id,
+        version: ellipse.version,
+        versionNonce: ellipse.versionNonce,
+      }),
+    ]);
+  });
+
+  it("bump versions of local duplicate elements when supplied (restoreElements)", () => {
```

Comment:
```suggestion
  it("bump versions of local duplicate elements when supplied"), () => {
```
Also please move this to inside `describe('restoreElements')`